app/actions/instancesManager.js

```diff
@@ -41,10 +41,16 @@ export function startInstance(instanceName) {
   return async (dispatch, getState) => {
     const { auth, settings } = getState();
 
+    const config = JSON.parse(
```

Comment:
The config file is already read inside the `launchCommand` function. Avoid reading it multiple times